.test-infra/metrics/build.gradle

```diff
@@ -123,9 +123,9 @@ task deploy {
       exec {
         executable 'sh'
         args '-c', "kubectl set image deployment/beamgrafana \
-          beamgrafana=gcr.io/apache-beam-testing/beamgrafana:${project.lastCommitId} \
-          beammetricssyncjenkins=gcr.io/apache-beam-testing/beammetricssyncjenkins:${project.lastCommitId} \
-          beammetricssyncgithub=gcr.io/apache-beam-testing/beammetricssyncgithub:${project.lastCommitId}"
+          beamgrafana-test=gcr.io/apache-beam-testing/beamgrafana-test:${project.lastCommitId} \
```

Comment:
Why are we retargeting these?

Reply:
Oh I need to change this one to the correct values.
For testing purposes we targeted it to another instance.
Thanks for the heads up!